Tests/Stream/Client/StreamWrapperTests.cs

```diff
@@ -69,13 +71,22 @@ public void SetUp()
     partition_ = configuration.GetValue<string>("Partition");
 
     Console.WriteLine($"endpoint : {options.Endpoint}");
-    var channel = GrpcChannelFactory.CreateChannel(options);
-    client_ = new Submitter.SubmitterClient(channel);
+    channel_ = GrpcChannelFactory.CreateChannel(options);
+    client_  = new Submitter.SubmitterClient(channel_);
     Console.WriteLine("Client created");
   }
 
+  [TearDown]
+  public void TearDown()
+  {
+    partition_ = null;
+    client_    = null;
```

Comment:
Do we know if `client_` is properly disposed ?

Reply:
There are no dispose function in it